draft-ietf-quic-tls.md

```diff
@@ -1197,6 +1197,64 @@ TLS ClientHello.  The server MAY retain these packets for later decryption in
 anticipation of receiving a ClientHello.
 
 
+## Retry Packet Integrity {#retry-integrity}
+
+Retry packets (see the Retry Packet section of {{QUIC-TRANSPORT}}) carry a
+Retry Integrity Tag that provides two properties: it allows discarding
+packets that have accidentally been corrupted by the network, and it mitigates
+off-path attackers' ability to send valid Retry packets.
```

Comment:
I'm unsure how this achieves the latter goal?

Reply:
We prevent off-path attackers from injecting Retry packets by including the Original Destination Connection ID in the associated data here. If they didn't see the client's Initial their Retry will not validate.

Reply:
Thanks, how about being being specific and saying "to send valid retry packets without having seen the original destination connection ID."?

That was also true before this change, so I was thinking there was some new protection I was unaware of.